publication-commons/src/test/java/no/unit/nva/publication/service/impl/DoiRequestServiceTest.java

```diff
@@ -0,0 +1,158 @@
+package no.unit.nva.publication.service.impl;
+
+import static org.hamcrest.CoreMatchers.containsString;
+import static org.hamcrest.CoreMatchers.equalTo;
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.core.Is.is;
+import static org.hamcrest.core.IsInstanceOf.instanceOf;
+import static org.hamcrest.core.IsNot.not;
+import static org.hamcrest.core.IsNull.nullValue;
+import static org.junit.jupiter.api.Assertions.assertDoesNotThrow;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
+import com.amazonaws.services.dynamodbv2.model.TransactionCanceledException;
+import java.io.IOException;
+import java.net.URI;
+import java.time.Clock;
+import java.time.Instant;
+import java.util.function.Supplier;
+import no.unit.nva.identifiers.SortableIdentifier;
+import no.unit.nva.model.Publication;
+import no.unit.nva.publication.PublicationGenerator;
+import no.unit.nva.publication.exception.InvalidPublicationException;
+import no.unit.nva.publication.service.ResourcesDynamoDbLocalTest;
+import no.unit.nva.publication.service.impl.exceptions.BadRequestException;
+import no.unit.nva.publication.storage.model.DoiRequest;
+import nva.commons.apigateway.exceptions.ConflictException;
+import nva.commons.apigateway.exceptions.NotFoundException;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.function.Executable;
+
+public class DoiRequestServiceTest extends ResourcesDynamoDbLocalTest {
+
+    public static final String NOT_THE_RESOURCE_OWNER = "someOther@owner.org";
+    public static final String SOME_USER = "some@user.com";
+    public static final URI SOME_PUBLISHER = URI.create("https://some-publicsher.com");
```

Comment:
some-publisher.example.org to ensure we know this is example data

Reply:
fixed